olp-cpp-sdk-dataservice-read/src/generated/api/PlatformApi.cpp

```diff
@@ -19,6 +19,9 @@
 
 #include "PlatformApi.h"
 
+#include <memory>
```

Comment:
Leftover from previous patch-set

Reply:
nevertheless, it's required for shared_ptr used in the file.